src/components/AddBoxForm.js

```diff
@@ -54,7 +54,14 @@ const AddBoxForm = ({ classes, onClose, onSubmit, products }) => (
                 margin="dense"
               >
                 {products.map(n => (
```

Comment:
I know was already there, but can you rename the `products.map(n => ...` to something like `products.map(prod => ...` :)